### DISCUSSION TITLE
DL image numerical target Question

### DISCUSSION THREAD

**Discussion by ghost** [satellite-image-deep-learning/techniques#33] [Q&A]
Hi there

Thanks so much for your repo. As a beginner, it has been instrumental for my learning.

I have a question regarding deep learning that I hope you can help me with.

I have numerical features (e.g. precipitation, temperature, slope) map as rasters (tif) files, and numerical target (e.g. soil moisture) map as raster tif files also. I need to use the futures to predict the target using DL (CNN or Unet). Most resouces I looked at only show DL with images for segmentation and classification where the target is CATEGORICAL (binary or multi classes) but is not numerical. I am struggling particularly with the following:

- How to build dataloarder 
- How to code the output layer after building the hidden layers of the network 

Could you please help with this? or point to resources (preferably, tutorials) that show how to tackle problems like this?

Big thanks in advance,

**Comment by robmarkcole** [comment]
I suggest to also establish a baseline using binary mask - i.e. high/low moisture per pixel. 

For a % per pixel see this approach https://github.com/tha-santacruz/BayesianUNet

Do let me know how you get on

**Reply by johnnqasim** [reply]
It seems like you're asking about setting a numerical target for a deep learning (DL) image task. Could you please provide more details or clarify your question? Are you looking to set a specific [메이저놀이터 ](http://jun2024.com/)performance goal for a DL model working on image data?

**Comment by johnqasim** [comment]
The advent of technology has revolutionized the way we entertain ourselves, and one of the most popular forms of online entertainment today is the online casino. With [온라인카지노](http://juncc2024.com) the rise of internet accessibility, players can now experience the thrill of a casino from the comfort of their own homes.
